### PR TITLE
Language / Animation Documentation

### PR DESCRIPTION
**Pull request by ryanbrownhill**
Relates to #276

### CODE REVIEW

**Comment by Snugug** [comment]
:+1: I think

**Comment by poofichu** [comment]
That's a toughie :-/

File: language/ui-patterns/01-documentation/animation/index.md [hunk 1]
```diff
@@ -47,24 +64,24 @@ When animating only a single property, follow the guidelines below.
...
+    * **Entrances** - `map-get($timing-functions, snap-in)` (Fast to Slow) 
+    * **Exits** - `map-get($timing-functions, snap-out)` (Slow to Fast)
+    * **Interactive Elements** - `map-get($timing-functions, snap-in-out)`
 
 ### Multiple Property Animations
 
-Multiple property animations are animations where multiple properties are being animated together. With multiple property animations there are two different options. The first option is to start one property alone, then animate any additional properties. The second option is for both properties to start at the same time, then have one property end before the other. We recommend not starting and stopping multiple properties at the same time; choose one option or the other.
+Multiple properties are animated together is known as a multiple property animation. With multiple property animations, there are two different options: to start one property alone then animate any additional properties, or to start both properties at the same time then have one property end before the other. We recommend not starting and stopping multiple properties at the same time; choose one option or the other.
```
Comment (bobbinrobyn) [inline language/ui-patterns/01-documentation/animation/index.md]: When more than one property is being animated, the animation is known as a multiple property animation.

**Comment by ryanbrownhill** [comment]
Ok now that travis passed :+1: 's everyone?

I will wait til @bobbinrobyn does a proofread pass to merge

**Comment by bobbinrobyn** [comment]
Okay, I just have two things:

Add a comma here --> "are performant, see"
Add a comma here --> "them animating, visit"

**Comment by bobbinrobyn** [comment]
Now that we've learned a little bit about dependent clauses, this gets my :+1: once those commas are added :)

**Comment by ryanbrownhill** [comment]
@IBM-Watson/design-acceleration :boy: says :green_heart: so this is ready for :+1: !

**Comment by bobbinrobyn** [comment]
:+1:

**Pull request merged**